src/components/accordion/accordion.tsx

```diff
@@ -41,7 +41,7 @@ export class Accordion {
    * Specifies the selection mode - "multi" (allow any number of open items), "single" (allow one open item),
    * or "single-persist" (allow and require one open item).
    */
-  @Prop({ reflect: true }) selectionMode: "multi" | "single" | "single-persist" = "multi";
+  @Prop({ reflect: true }) selectionMode: AccordionSelectionMode = "multiple";
```

Comment:
Do you think we should leave the defaults to avoid breaking users that are querying or styling via the attribute?

Reply:
I think that's a good idea, considering users may apply styles based on attribute values.